lib/modules/filteReddit.js

```diff
@@ -64,6 +67,11 @@ module.options = {
 		value: false,
 		description: 'Don\'t filter anything on users\' profile pages',
 	},
+	showRemoveVisitedTab: {
+		type: 'boolean',
+		value: true,
+		description: `${process.env.BUILD_TARGET === 'safari' ? '(Not available on Safari) ' : ''}Show a \'remove visited\' tab at the top of each subreddit, to easily toggle whether to remove visited links.`,
```

Comment:
Use `value: process.env.BUILD_TARGET !== 'safari'` and `noconfig: process.env.BUILD_TARGET === 'safari'`